Data Structures/Linked-Lists/Circular Linked List/deletion.cpp

```diff
@@ -0,0 +1,52 @@
+/* Deletes a node of given address in a circular linked list where last node's next points to first node.
```

Comment:
Every file should have the main function. you can combine insertion and deletion in one file.